test/state/state.cpp

```diff
@@ -177,10 +180,7 @@ std::variant<TransactionReceipt, std::error_code> transition(
     state.touch(block.coinbase).balance += gas_used * priority_gas_price;
 
     // Apply destructs and clear erasable empty accounts.
```

Comment:
I'd move this comment to the definition of `finalize`.

Reply:
We already have some doc comment there https://github.com/ethereum/evmone/pull/609/files#diff-5e1562f74d0f9de52fe46c9a338ee6cdbbf9b573aaac29f4232a4d8183eac44f
I can add this one too,.